control/bdalg.py

```diff
@@ -60,14 +60,16 @@
 
 __all__ = ['series', 'parallel', 'negate', 'feedback', 'append', 'connect']
 
-def series(sys1, sys2):
-    """Return the series connection sys2 * sys1 for --> sys1 --> sys2 -->.
+def series(sys1, sys2,*sysn):
+    """Return the series connection (... * sys3 *) sys2 * sys1 
```

Comment:
Yes, it's  a good idea. But, using functional programming would increase the complexity of the code or decrease the elegance of the code.